bucket/git-credential-manager-core.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.475",
+    "description": "Git Credential Manager Core (GCM Core) is a secure Git credential helper built on .NET that runs on Windows and macOS. Linux support is in an early preview.",
+    "homepage": "https://github.com/microsoft/Git-Credential-Manager-Core",
+    "license": "MIT",
+    "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v2.0.475/gcmcore-win-x86-2.0.475.64295.zip",
+    "hash": "498069454e7759364dd1c168a25d1f593d9295db521d4ec33e5897ce1344ea68",
+    "bin": "git-credential-manager-core.exe",
+    "shortcuts": [
+        [
+            "git-credential-manager-core.exe",
+            "GCM"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/microsoft/Git-Credential-Manager-Core",
+        "regex": "/releases/tag/v([\\d.]+)"
```

Comment:
Use full version instead of partial tag name

Reply:
how to do that? I based off an application submitted earlier. 

here is the path from the source repo
https://github.com/microsoft/Git-Credential-Manager-Core/releases/tag/v2.0.475

Reply:
@parthopdas 
If it helps, you may want to take a look at https://raw.githubusercontent.com/AntonOks/scoop-aoks/master/bucket/Git-Credential-Manager-Core-aoks.json
Not saying, this would be approved / committed, but at least it worked with the last tool updates... and it may give to an idea how to get your manifest finally merged...

Reply:
@parthopdas
As far as I know. The `18224` in `gcmcore-win-x86-2.0.567.18224.zip` is the **build number**. (This is the convention of many Microsoft products)

Therefore, we should include build number in the version string. The "full version number" mentioned by Ash is `2.0.567.18224`. This help us to keep track of all the minor updates.

For details about the variables, check: https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate

```
    "checkver": {
        "github": "https://github.com/microsoft/Git-Credential-Manager-Core",
        "regex": "gcmcore-win-x86-([\\d.]+)\\.zip"
    },
    "autoupdate": {
        "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v$matchHead/gcmcore-win-x86-$version.zip"
    }
```